### ISSUE TITLE
wp_localize_script $l10n should be a array in wordpress 5.7.0

### ISSUE THREAD

**Issue by LucianoHanna** [kasparsd/code-prettify#35]
There's a warning about wp_localize_script used in code-prettify.php:46, third parameter should be an array

```php
wp_localize_script(
	'code-prettify',
	'codePrettifyLoaderBaseUrl',
	plugins_url( 'prettify', __FILE__ )
); 
```

**Comment by cgarofalo** [comment]
Any updates on this issue?

**Comment by kasparsd** [comment]
This has been fixed in version 1.5.1.
